channeld/channeld.c

```diff
@@ -2915,6 +2915,64 @@ static size_t calc_weight(enum tx_role role, const struct wally_psbt *psbt)
 	return weight;
 }
 
+/* Get the remote node amount after the splice */
+static struct amount_msat remote_balance(struct peer *peer,
+					    enum tx_role our_role,
+					    const struct wally_psbt *psbt,
+					    int chan_output_index,
+					    int chan_input_index)
+{
+	s64 relative_remote_val;
+	struct amount_msat push_value;
+	struct amount_msat in[NUM_TX_ROLES], out[NUM_TX_ROLES];
+	const bool opener = our_role == TX_INITIATOR;
+
+	/* FIXME(vincenzopalazzo): move this in another method and resuse it in different places */
+	in[TX_INITIATOR] = peer->channel->view->owed[opener ? LOCAL : REMOTE];
+	in[TX_ACCEPTER] = peer->channel->view->owed[opener ? REMOTE : LOCAL];
```

Comment:
this is computed, but then unused

Reply:
also, I don't understand what is the overall logic.  could you comment in the PR or in the code what is the formula that is being calculated?

Reply:
You are correct, we have the channel amount before splice inside `peer->channel->view->owed[opener ? REMOTE : LOCAL];` and the relative splice amount inside `peer->splice->*_relative` so all this code can be semplified.